src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,93 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class Users {
```

Comment:
As a best practice in OOP, entity classes should  be named as singular
```suggestion
public class User {
```